crates/viewer/re_viewer/src/ui/settings_screen.rs

```diff
@@ -204,40 +204,39 @@ fn video_section_ui(ui: &mut Ui, app_options: &mut AppOptions) {
 
 #[cfg(not(target_arch = "wasm32"))]
 fn ffmpeg_path_status_ui(ui: &mut Ui, app_options: &AppOptions) {
+    use re_ui::ContextExt as _;
     use re_video::decode::{FFmpegVersion, FFmpegVersionParseError};
 
     let path = app_options
         .video_decoder_override_ffmpeg_path
         .then(|| std::path::Path::new(&app_options.video_decoder_ffmpeg_path));
 
-    if let Some(path) = path {
-        if !path.is_file() {
-            ui.error_label("The specified FFmpeg binary path does not exist or is not a file.");
-            return;
-        }
-    }
-
-    let res = FFmpegVersion::for_executable(path);
-    match res {
-        Ok(version) => {
-            if version.is_compatible() {
-                ui.success_label(&format!("FFmpeg found (version {version})"));
-            } else {
-                ui.error_label(&format!("Incompatible FFmpeg version: {version}"));
+    let ctx = ui.ctx();
+    let label_text = if path.is_some_and(|path| !path.is_file()) {
+        ctx.error_text("The specified FFmpeg binary path does not exist or is not a file.")
+    } else {
+        let res = FFmpegVersion::for_executable(path);
+
+        match res {
+            Ok(version) => {
+                if version.is_compatible() {
+                    ctx.success_text(format!("FFmpeg found (version {version})"))
+                } else {
+                    ctx.error_text(format!("Incompatible FFmpeg version: {version}"))
```

Comment:
The box version might be good here actually for warning & error if it wasn't for the discrepancy with the success text.
That's ofc only if one wouldn't add a success box as well :smile:

Reply:
Yeah, it would require a success-label with border then… and icon. Otherwise the text moves with what you type.

Reply:
something like ✔️  or ✅